fixtures/enum-types/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+enum Animal {
+    Dog,
+    Cat,
+}
+
+#[repr(u8)]
```

Comment:
If we are going to have a whole fixture for this, we might as well exercise more types, so maybe an enum with a values but no repr, an explicit signed repr (eg, i8), and an enum with a, say, usize repr (which you'd probably use an unsigned int in bidnings? )Even if some of them turn out to be problematic and ends up being done in a followup it might be worth having the basics here with a link to followup issues or similar?